### DISCUSSION TITLE
server output in ansi (windows-1252) only

### DISCUSSION THREAD

**Discussion by NotSignificant** [dbeaver/dbeaver#20579] [Q&A] [answered]
I don't know if this was asked already or if it's known (I didn't find anything). I tryed my best - even with tips from this source here - to get all outputs in dbeaver to utf8 (like all my databases do). This works well in all cases but not in the server output window (SQL editor bottom toolbar), where still windows-1252 is used. Did I miss something (all my dbeaver preferences seem to be configured as utf8) or is it a bug?
![anhang](https://github.com/dbeaver/dbeaver/assets/133522530/8100a245-bc81-4367-918c-900624d280be)
these cryptic letters should be the euro sign.

I'm using dbeaver within the eclipse ide

**Comment by E1izabeth** [comment]
What database do you use?

**Comment by NotSignificant** [comment]
I'm using PostgreSQL and MySQL and in both the same behaviour, but only in the server outputs. All DBs are configured in utf8.

**Comment by E1izabeth** [comment]
Probably, it's related to issues #9890 and/or #19631

**Comment by NotSignificant** [comment]
Sorry, but no. The display of the result-sets seem to be okay, but I don't have checked CJK-Codes. The only output, that is weired, is the server output, where "raise notice" outputs and such things are shown. It seems to be like a console output, but there isn't any preference to configure it.
![result_out](https://github.com/dbeaver/dbeaver/assets/133522530/304fc78a-a348-4304-b3b0-5ed918b88390)

**Comment by E1izabeth** [comment]
I see. Thank you for explanation.
Could you create an issue about it?

**Comment by NotSignificant** [comment]
I don't know how to do that. :(
I'm not firm in githubbing.

**Reply by E1izabeth** [reply]
https://github.com/dbeaver/dbeaver/issues/new/choose

**Comment by NotSignificant** [comment]
Done... hope it's done correctly.

**Comment by E1izabeth** [comment, marked as answer]
Thank you. I hope we will figure out what's the issue cause and solve it.

**Comment by NotSignificant** [comment]
Okay, then you can close this discussion, thanks.
